Makefile

```diff
@@ -1176,6 +1176,9 @@ endif
 	pkgbuild --version $(FULLVERSION) \
 		--identifier org.nodejs.node.pkg \
 		--root $(MACOSOUTDIR)/dist/node $(MACOSOUTDIR)/pkgs/node-$(FULLVERSION).pkg
+	mkdir -p $(MACOSOUTDIR)/dist/npm/usr/local/bin
+	ln -sf ../lib/node_modules/npm/bin/npm-cli.js $(MACOSOUTDIR)/dist/npm/usr/local/bin/npm
+	ln -sf ../lib/node_modules/npm/bin/npx-cli.js $(MACOSOUTDIR)/dist/npm/usr/local/bin/npx
```

Comment:
That doesn't seem correct, we don't want to install those on the release machine
```suggestion
```

Reply:
You can either place these files on the release machine before invoking the `pkgbuild` and then the are inside and no postinstall is needed or you need the postinstall script.

One could remove them right after the `pkgbuild` though.

Reply:
I am also unsure what you mean by "release machine".

These symlinks are created inside the build folder `out` which is gitignored.

Reply:
Moving the `unlink` call to after invocating `pkgbuild` sounds like a better solution

Reply:
```shell
unlink $(MACOSOUTDIR)/dist/node/usr/local/bin/npx
```

The `unlink`s are for the `npm` and `npx` symlinks in the `node` and not the `npm` directory; these would have been placed in the `org.nodejs.node.pkg` package. 

The symlinks created by this PR will be placed in the `org.nodejs.npm.pkg` package.

One could add 2 additional `unlink` statements for `npm` and `npx` after the `pkgbuild` though.

But I am not sure if it is worth it because aren't these links on an ephemeral directory on a build machine?